docs/sbpy/spectroscopy.rst

```diff
@@ -6,6 +6,138 @@ Introduction
 
 `~sbpy.spectroscopy` provides routines for the modeling and analysis of emission spectra of gas in comets and reflection spectra of asteroid surfaces.  Sub-modules `sun` and `vega` control `sbpy`'s photometric calibration.
 
+JPLSpec Constants and Line Intensity Integral Conversion
+--------------------------------------------------------
+
+`sbpy.spectroscopy` has a function called ``molecular_data`` which takes care of
+querying the JPL Molecular Spectral Catalog through the use of
+`astroquery.jplspec` and calculates all the necessary constants needed for both
+production rate and Einstein coefficient calculations. The function
+``intensity_conversion`` takes care of converting the intensity line integral at
+300 K found in JPL Spec catalog and convert it closer to the temperature given
+by the user.
+
+.. code-block:: python
+
+   >>> from sbpy.spectrosocpy import molecular_data, intensity_conversion
```

Comment:
there is typo leading to the failed build: `from sbpy.spectrosocpy...` -> `from sbpy.spectroscopy...`

Reply:
Thank you, I missed this.